### DISCUSSION TITLE
Logging queries

### DISCUSSION THREAD

**Discussion by uzdik-student** [ClickHouse/clickhouse-go#1323] [Q&A] [answered]
For quick debugging, we need to print executed queries to stdout. How to do it?

**Comment by jkaflik** [comment, marked as answer]
@arman-sydikov 

`Debug` and `Debugf` connection options could be useful: https://github.com/ClickHouse/clickhouse-go/blob/eec85ecc060c78404c8305ec02a11a0f1cb3892c/examples/clickhouse_api/connect_settings.go#L47-L49
